### ISSUE TITLE
Circadian lighting stopped working

### ISSUE THREAD

**Issue by His-Holy-Noodliness** [claytonjn/hass-circadian_lighting#164]
Hi,

I have been using CL for about a year with few issues (well, initial setup was a little complex, since I put in place override scripts, but nonetheless, it's been working very reliably) but has suddenly stopped working.

A few weeks ago, due to some Astral version bump or something, I recall having to add manual sunset_time and sunrise_time to my config, which temporarily fixed that, then suddenly a few days ago my circadian lights stopped working again, and this time, even rolling back versions isn't helping - I can't get them working again.

My configuration is like this:

circadian_lighting:
  min_colortemp: 2500
  max_colortemp: 5500
#  sunrise_time: '07:20:00'
#  sunset_time: '20:45:00'
switch: !include circadian_switches.yaml

As you can see I have commented back out the sunrise and sunset times since the Astral problem seems fixed, but even putting them back in uncommented doesn't fix the new problem.

The included circadian_switches.yaml then contains numerous stanzas like this:

numerous zones, all more or less like:

  - platform: circadian_lighting
    name: Lounge Ceiling
    min_brightness: 45
    sleep_entity: 'input_boolean.sleep'
    sleep_state: 'on'
    sleep_colortemp: 2200
    sleep_brightness: 10
    lights_ct:
      - light.lounge_ceiling_lights
    lights_brightness:
      - light.lounge_ceiling_lights

All seem to be correct (well, nothing had changed anyway) and are creating the corresponding switch.circadian_lighting_zone entity. It just doesn't do anything any more!

Home Assistant is the managed/supervisor install in a Proxmox VM, unaltered from the official HA appliance.

I've tried enabling logging, and it looks like circadian lighting _does_ update the lights, but with the wrong values:

2021-07-07 20:46:26 DEBUG (MainThread) [custom_components.circadian_lighting.switch] Lights adjusting because light.lounge_ceiling_lights changed from off to on and these attributes changes: color_mode: hs, brightness: 254, hs_color: (36.279, 67.451), rgb_color: (255, 186, 82), xy_color: (0.506, 0.414), effect: none.
2021-07-07 20:46:26 DEBUG (MainThread) [custom_components.circadian_lighting.switch] Scheduling 'light.turn_on' with the following 'service_data': {'entity_id': 'light.lounge_ceiling_lights', 'transition': 1.0, 'brightness': 254}

2021-07-07 20:46:26 DEBUG (MainThread) [custom_components.circadian_lighting.switch] Lights adjusting because light.dining_room_lights changed from off to on and these attributes changes: color_mode: color_temp, brightness: 206, color_temp: 379.
2021-07-07 20:46:26 DEBUG (MainThread) [custom_components.circadian_lighting.switch] Scheduling 'light.turn_on' with the following 'service_data': {'entity_id': 'light.dining_room_lights', 'transition': 1.0, 'brightness': 254}

I might be wrong, but that looks like it's trying to maximise the brightness of those lights (and not set any colour temperature), however nothing actually happens to those lights - for example, if I set them to dim red, they don't then update and turn white, or even increase in brightness, they just stay unchanged. 

I can manually control them from within HA, set a color temp, brightness or colour and everything works as it should.

The bulbs are all genuine Philips Hue, through genuine Hue hubs using the Hue integration.

Finally, the fact that the integration seems to be trying to set maximum brightness in the evening seems even odder because the circadian values sensor is updating correctly - see below.

Circadian Values
3 minutes ago
4.33293795424098 %
No state history found.
Attributes
Colortemp
2629.988138627229
Rgb color
255, 164.1066297080045, 81.58005748706091
Xy color
0.532, 0.388

Any ideas?

**Comment by claytonjn** [comment]
The formatting of your configs got a bit messed up, you should wrap them in ``` so that GitHub doesn't think you're using markdown in those sections. 

My initial assumption would be that the issue is because in the example you posted it looks like you have the light listed under `lights_ct` and `lights_brightness`. Can you explain what your intention was with setting up the configuration that way?

**Comment by His-Holy-Noodliness** [comment]
Wow, that's pretty strange, I just re-read the github after reading your reply, and it seems something has changed in a recent update, because I have always had it configured that way (with both lights_ct and lights_brightness) under the mistaken understanding that the two aspects were adjusted separately and that you listed lights in both if you wanted both aspects updated.

It seems that strange configuration, which previously worked, is now broken, so perhaps I'm the only one that thought you needed to have it configured that way!

I changed a zone to only lights_ct and it immediately started working, so I guess I just need to change them all now.

**Comment by His-Holy-Noodliness** [comment]
And by the way, I should take this opportunity to thank you for the dev work - I've done bits in the past, so I know it can be a pain. It's only when this stuff goes wrong that you realise how much it improves your life - thanks for a really nice integration.

**Comment by claytonjn** [comment]
If I had to guess, without spending time looking into it, I would guess that the change in behavior was caused by the switch to async and the refactoring that went along with that. 

I would like to tie into the configuration checking and have it throw an error if an entity is listed in more than one of the `lights_...` sections, but I haven't had time to look into how to do that if it's even possible. I'm glad it's working again for you though!
